common/buildcraft/builders/tile/TileBuilder.java

```diff
@@ -246,6 +247,7 @@ public void writePayload(int id, PacketBufferBC buffer, Side side) {
         }
     }
 
+    @SideOnly(Side.CLIENT)
```

Comment:
it uses a client side only method still, tankManager.readData

i'll remove the annotation but if this gets called on a server it will crash on that then